tests/sav/nitg-e/test_json_deserialization_alt1.res

```diff
@@ -1,40 +1,40 @@
-Runtime error: Aborted (../lib/serialization/serialization.nit:109)
+Runtime error: Aborted (../lib/serialization/serialization.nit:111)
 # Nit:
-<A: true a 0.123 1234 asdf false>
+<A: true a 0.123 1234 asdf false p4ssw0rd>
```

Comment:
there is a lot of serialized passwords the the .res files. Is this expected?

Maybe you should also check [this page](https://help.github.com/articles/remove-sensitive-data/) :)